README.md

```diff
@@ -58,7 +30,6 @@ After you're finished please remove all the comments and instructions!
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
 # 📖 [Island Blogs] <a name="about-project"></a>
```

Comment:
- [OPTIONAL] Kindly remove  React.js in the Client Tech Stack since we're not using it in the project 👇🏻 

![image](https://user-images.githubusercontent.com/79058364/228728863-c6adad88-e590-4283-a719-c96dfde357b7.png)